packages/phone-number-privacy/signer/src/salt-generation/get-salt.ts

```diff
@@ -33,7 +36,7 @@ export async function handleGetBlindedMessageForSalt(
 ) {
   logger.info('Begin getBlindedSalt request')
   try {
-    if (!isValidGetSignatureInput(request.body)) {
+    if (!isValidGetSignatureInput(request.body) || (await getRequestExists(request.body))) {
       respondWithError(response, 400, WarningMessage.INVALID_INPUT)
```

Comment:
I think we should let this request go through and just not consume quota. If the response got dropped on the network, the client or combiner should retry with the same request timestamp in order to not consume quota unnecessarily.